source/features/remove-label-faster.tsx

```diff
@@ -0,0 +1,85 @@
+import './remove-label-faster.css';
+import React from 'dom-chef';
+import XIcon from 'octicon/x.svg';
+import select from 'select-dom';
+import oneTime from 'onetime';
+import delegate from 'delegate-it';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+
+import {getRepoURL, getConversationNumber} from '../github-helpers';
+
+const canNotEditLabels = oneTime((): boolean => !select.exists('.sidebar-labels .octicon-gear'));
+
+function updateSidebar() {
+	select('#partial-discussion-sidebar')!.dispatchEvent(new CustomEvent('socket:message', {
+		detail: {
+			name: '',
+			data: {},
+			cached: false
+		}
+	}));
+}
+
+async function removeLabelButtonClickHandler(event: delegate.Event<MouseEvent, HTMLButtonElement>): Promise<void> {
+	event.preventDefault();
+
+	const removeLabelButton = event.delegateTarget;
+
+	removeLabelButton.disabled = true;
+	await api.v3(`repos/${getRepoURL()}/issues/${getConversationNumber()!}/labels/${removeLabelButton.dataset.name!}`, {
+		method: 'DELETE'
+	});
+
+	updateSidebar();
```

Comment:
This doesn't seem to work reliably, especially in Firefox.

Let's just remove the label, it's faster anyway
```suggestion
	removeLabelButton.closest('a')!.remove()
```